1.7: build fails

gcc 12.0.0.
```console
[tkloczko@ss-desktop x86_64-redhat-linux-gnu]$ ninja -k 0
[16/116] Compiling C object swaynag/swaynag.p/swaynag.c.o
FAILED: swaynag/swaynag.p/swaynag.c.o
/usr/bin/gcc -Iswaynag/swaynag.p -Iswaynag -I../swaynag -Iinclude -I../include -Iprotocols -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/harfbuzz -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Werror -std=c11 -DWLR_USE_UNSTABLE -Wno-unused-parameter -Wno-unused-result -Wno-missing-braces -Wundef -Wvla '-DSYSCONFDIR="//etc"' '-DSWAY_VERSION="1.7"' -fmacro-prefix-map=../= -O2 -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fdata-sections -ffunction-sections -flto=auto -flto-partition=none -MD -MQ swaynag/swaynag.p/swaynag.c.o -MF swaynag/swaynag.p/swaynag.c.o.d -o swaynag/swaynag.p/swaynag.c.o -c ../swaynag/swaynag.c
../swaynag/swaynag.c: In function ‘swaynag_destroy’:
../swaynag/swaynag.c:514:13: error: the comparison will always evaluate as ‘true’ for the address of ‘buffers’ will never be NULL [-Werror=address]
  514 |         if (&swaynag->buffers[0]) {
      |             ^
In file included from ../include/swaynag/render.h:3,
                 from ../swaynag/swaynag.c:11:
../include/swaynag/swaynag.h:92:28: note: ‘buffers’ declared here
   92 |         struct pool_buffer buffers[2];
      |                            ^~~~~~~
../swaynag/swaynag.c:518:13: error: the comparison will always evaluate as ‘true’ for the address of ‘buffers’ will never be NULL [-Werror=address]
  518 |         if (&swaynag->buffers[1]) {
      |             ^
In file included from ../include/swaynag/render.h:3,
                 from ../swaynag/swaynag.c:11:
../include/swaynag/swaynag.h:92:28: note: ‘buffers’ declared here
   92 |         struct pool_buffer buffers[2];
      |                            ^~~~~~~
cc1: all warnings being treated as errors
[115/116] Linking target sway/sway
ninja: build stopped: cannot make progress due to previous errors.
```

--- a/swaynag/swaynag.c
+++ b/swaynag/swaynag.c
@@ -511,13 +511,8 @@ void swaynag_destroy(struct swaynag *swaynag) {
 		swaynag_seat_destroy(seat);
 	}
 
-	if (&swaynag->buffers[0]) {
-		destroy_buffer(&swaynag->buffers[0]);
-	}
-
-	if (&swaynag->buffers[1]) {
-		destroy_buffer(&swaynag->buffers[1]);
-	}
+	destroy_buffer(&swaynag->buffers[0]);
+	destroy_buffer(&swaynag->buffers[1]);
 
 	if (swaynag->outputs.prev || swaynag->outputs.next) {
 		struct swaynag_output *output, *temp;

